### DISCUSSION TITLE
Help to config rqlite for my app.

### DISCUSSION THREAD

**Discussion by midas95** [rqlite/rqlite#1668] [Q&A] [answered]
Hi, everyone.
I am working on data sync desktop app. The app will sync end-user's sqlite database. so I would like to use rqlite that end user save their databases.In this app, users save their database on rqlite backend and restore and backup their database to backend.
Can i make that system using rqlite? if yes, where can i start? I would like to looking for some examples for main features.
Thanks in advance.

**Comment by otoolep** [comment]
What have you tried already? How end-users do want to support? Do you know rqlite is not a drop in replace for SQLite? Have you reviewed the rqlite docs at https://rqlite.io/docs/?

That said, rqlite is usually not a good fit for simply sync'ing SQLite databases. That is not it's purpose, really. You should take a look at the [rqlite FAQ](https://rqlite.io/docs/faq/).

**Comment by midas95** [comment]
Thanks for your reply.

**Comment by midas95** [comment]
Hi, otoolep.
"rqlite is about replicating a set of data, which has been written to it using SQL. The data is replicated for fault tolerance because your data is so important that you want multiple copies distributed in different places, you want be able to query your data even if some machines fail, or both. These different places could be different machines on a rack, or different machines, each in different buildings, or even different machines, [each on different continents](https://www.philipotoole.com/rqlite-v3-0-1-globally-replicating-sqlite/)."

I want users save database safely and allow users restore and sync database like git.
I want to know availability to implement that functionality. Looking forward to hearing from you soon.

**Reply by otoolep** [reply]
What is your application? I think you want something more like [litestream](https://litestream.io/), not rqlite.

**Comment by otoolep** [comment, marked as answer]
rqlite is a highly available relational database, which can be used to store critical data. It can be run as a cluster, allowing queries to be served even if individual nodes fail. rqlite is not just, however, a backup solution allowing users to simply replicate their SQLite data to another place, allowing it to be restored again if needed. 

https://rqlite.io/docs/faq/#can-i-use-rqlite-to-replicate-my-sqlite-database-to-a-second-node

**Comment by midas95** [comment]
Thanks for reply.
